inbm/dispatcher-agent/dispatcher/dispatcher_class.py

```diff
@@ -247,7 +247,7 @@ def _perform_cmd_type_operation(self, parsed_head: XmlHandler, xml: str) -> Resu
             message = self.device_manager.restart()
             if message == SUCCESS_RESTART:
                 state: dispatcher_state.DispatcherState = {'restart_reason': 'restart_cmd'}
-                dispatcher_state.write_dispatcher_state_to_state_file(state)                
+                dispatcher_state.write_dispatcher_state_to_state_file(state)
```

Comment:
<h3>$\color{#FFD700}Coding Style: CodeConsistency$</h3>

Ensure that the added whitespace does not affect the code's logic or readability and adheres to the project's coding standards.

Reply:
out of scope--this is from reformatting the code base

also, trailing whitespace can always safely be removed in Python